test/unit/connection/test-connection-ssl-min-tls1.2-accept.js

```diff
@@ -0,0 +1,27 @@
+var assert     = require('assert');
+var common     = require('../../common');
+var connection = common.createConnection({
+  port : common.fakeServerPort,
+  ssl  : {
+    ca         : common.getSSLConfig().ca,
+    minVersion : 'TLSv1.2'
+  }
+});
+
+var server = common.createFakeServer();
+
+server.listen(common.fakeServerPort, function(err) {
+  if (err) throw err;
+
+  connection.ping(function(err) {
+    assert.ifError(err);
```

Comment:
Ideally the test should assert that it is failing for the _reason_ we expect it to. Found quite a few bugs a while back when I added those kinds of tests in this lib.

Reply:
Since this is happy path test do we need to check the error message? Could fail for any number of reason, for happy path the test doesnt need to know why right?